src/test/kotlin/com/zepben/evolve/cim/iec61968/infiec61968/infcommon/RatioTest.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61968.infiec61968.infcommon
+
+import com.zepben.testutils.exception.ExpectException.expect
+import com.zepben.testutils.junit.SystemLogExtension
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+
+internal class RatioTest {
+
+    @JvmField
+    @RegisterExtension
+    var systemErr: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+
+    @Test
+    internal fun accessorCoverage() {
+        val ratio = Ratio(9.0, 6.0)
+
+        assertThat(ratio.numerator, equalTo(9.0))
+        assertThat(ratio.denominator, equalTo(6.0))
+        assertThat(ratio.quotient, equalTo(1.5))
+    }
+
+    @Test
+    internal fun throwsExceptionOnZeroDenominator() {
+        val invalidRatio = Ratio(42.0, 0.0)
+        expect { invalidRatio.quotient }
+            .toThrow(IllegalArgumentException::class.java)
```

Comment:
suggestion: If you update the zepben test utils dependency to v1.2.0 you pick up some more Kotlin support and can pass the type via `.toThrow<IllegalArgumentException>()`